app/build.gradle

```diff
@@ -45,10 +45,10 @@ android {
 
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
-    implementation"org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
```

Comment:
" or ' I recommend you use the same syntax

Reply:
I understand but double quotes is for $variable interpolation.

Reply:
i will resolve this conversation :)